src/test/java/uk/gov/companieshouse/authcodenotification/controller/AuthCodeNotificationControllerTest.java

```diff
@@ -30,17 +31,20 @@ class AuthCodeNotificationControllerTest {
     @Mock
     private DataSanitisation dataSanitisation;
 
+
     @Test
     void testSendEmailReturnsSuccess() {
-        ResponseEntity<Object> responseEntity = controller.sendEmail(REQUEST_ID, COMPANY_NUMBER);
+        SendEmailRequestDto sendEmailRequestDto = new SendEmailRequestDto();
```

Comment:
define the SendEmailRequestDto at class level and add a beforeEach method to create the new dto, looks like we'll need it for each test

Reply:
Done this for two test classes.